src/main/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+#spring:
+#  datasource:
+#    username:
```

Comment:
numele dbului

Reply:
https://www.codejava.net/frameworks/spring-boot/connect-to-postgresql-database-examples